docs/release-notes.md

```diff
@@ -21,7 +21,7 @@ Minor changes:
 
 Internal changes:
 - rdcore: Rename `--kargs` to `--append-karg` in `zipl` subcommand and support passing it multiple times
-
+- io: open bls configs 'ro' by default
```

Comment:
`io` is only meaningful if one know the codebase well. Maybe something like:

> Only open BLS configs in write mode when modifying

?

Reply:
Also, can you add some background for this change in the commit message, maybe with a link to the dm-verity cosa PR?